contrib/TestHarness/Program.cs

```diff
@@ -1089,10 +1101,20 @@ private static void AppendToErrorSummary(string summaryFileName, List<XElement>
 
         private static void AppendToSummary(string summaryFileName, XElement xout, bool traceToStdout = false, bool shouldLock = true)
         {
+            bool useXml = true;
+            if (summaryFileName != null && summaryFileName.EndsWith(".json")) {

```

Comment:
This line was missing the first null check which originally caused Joshua unhappy.